include/ginkgo/core/solver/lower_trs.hpp

```diff
@@ -113,6 +103,13 @@ class LowerTrs : public EnableLinOp<LowerTrs<ValueType, IndexType>>,
         std::shared_ptr<const LinOpFactory> GKO_FACTORY_PARAMETER(
             preconditioner, nullptr);
 
+        /**
+         * Already generated preconditioner. If one is provided, the factory
```

Comment:
This should be removed.

Reply:
Thanks, I definitively missed to remove that.